docs/providers/aws/events/apigateway.md

```diff
@@ -730,6 +730,47 @@ functions:
               application/json: ${file(create_request.json)}
 ```
 
+API Gateway models can be customized by inline properties
+
+```yml
+functions:
+  create:
+    handler: posts.create
+    events:
+      - http:
+          path: posts/create
+          method: post
+          requestSchema:
+            application/json:
+              definition: ${file(create_request.json)}
+              name: PostCreateModel
+              description: 'Validation model for Creating Posts'
+```
+
+API Gateway models can be references by global models in the provider. The same structure exist
+for models defined in the provider as inline functions. Provider models default to `application/json`
+
+```yml
+provider:
+    ...
+    apiGateway:
+      requestSchemas:
+        post-create-model:
+          name: PostCreateModel
+          schema: ${file(api_schema/post_add_schema.json)}
```

Comment:
I guess for consistency it should also be named `definition`

Reply:
That was my fault. It is supposed to be `schema`